src/elements/toast/toast.visual.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { html, nothing } from 'lit';
+
+import { describeEach, describeViewports, visualDiffDefault } from '../core/testing/private.js';
+
+import './toast.js';
+import '../button.js';
+import '../link.js';
+
+describe(`sbb-toast`, () => {
+  const cases = {
+    icon: [false, true],
+    action: ['dismissible', 'button', 'link'],
+  };
+
+  const positionCases = [
+    'top-start',
+    'top-center',
+    'top-end',
+    'bottom-start',
+    'bottom-center',
+    'bottom-end',
+  ];
+
+  describeViewports({ viewports: ['zero', 'medium'], viewportHeight: 300 }, () => {
```

Comment:
I see some zero screenshots not present. is there any error present?

Reply:
e.g. `https://lyne-components-visual-regression-diff-pr2883.app.sbb.ch/compare/sbb-toast/icon=false-action=button_`

Reply:
I believe the empty name is the cause of it. I'll have a look and see if we can easily fix it

 I just wanted to avoid a useless 'default' :/

Reply:
I forgot an await :| 
I was unfair to blame the test name